.github/workflows/publish-auto.yml

```diff
@@ -26,7 +26,8 @@ jobs:
         run: yarn --frozen-lockfile
 
       - name: Create release
+        run: npm run release
```

Comment:
Wondering if using `yarn run release` might have better results than `npm run release`, noticed we are seeing the cleanup error in CI again https://github.com/vega/ts-json-schema-generator/runs/4232722268?check_suite_focus=true

Reply:
Let's try in https://github.com/vega/ts-json-schema-generator/pull/1017/commits/f5848fa95efb624c89ec06ab2a207296e1bc211f

Reply:
That fails: https://github.com/vega/ts-json-schema-generator/runs/4269550305?check_suite_focus=true